docs/blog/posts/introduction-to-bpfd.md

```diff
@@ -0,0 +1,342 @@
+---
+date: 2023-07-16
+authors:
+  - anfredette
+  - shaneutt
+---
+
+# Introduction To Bpfd
+
+In today's cloud ecosystem there's a high demand for low-level system access
+to enable high performance observability, security and networking functionality
+for applications. Historically a great deal of this kind of functionality has
+been implemented in [userspace], but the ability to program these kinds of
+things directly into the operating system can provide many benefits including
+(but not limited to) performance. However, it has historically been very
+challenging to add functionality directly to the operating system. In the past
+you may have ended up developing and managing cumbersome [kernel modules][kmod],
+but in recent years [eBPF] has emerged as a technology in the
+[Linux Kernel][linux] looking to change all that.
+
```

Comment:
Add:

`<!-- more -->`

This will render the first paragraph as the excerpt and render the Continue Reading button after it.

Reply:
Done.